packages/peregrine/lib/talons/RootComponents/Category/useCategoryContent.js

```diff
@@ -68,8 +89,33 @@ export const useCategoryContent = props => {
     const categoryDescription = categoryData
         ? categoryData.category.description
         : null;
+    const sortingMethods = sortData ? sortData.products.sort_fields.options : null;
+    const sortingMethodsDirections = sortingMethods ? sortingMethods.map(method => {
+        const sortingMethodAscending = {
+            id: `sortItem.${method.value}Asc`,
+            text: `${method.label} Asc`,
+            attribute: method.value,
+            sortDirection: 'ASC'
+        };
+        const sortingMethodDescending = {
+            id: `sortItem.${method.value}Desc`,
+            text: `${method.label} Desc`,
+            attribute: method.value,
+            sortDirection: 'DESC'
+        }
```

Comment:
If you look in `productSort.js`, there is this default props object for sorting:

```javascript
availableSortMethods: [
        {
            id: 'sortItem.position',
            text: 'Position',
            attribute: 'position',
            sortDirection: 'ASC'
        },
        {
            id: 'sortItem.relevance',
            text: 'Best Match',
            attribute: 'relevance',
            sortDirection: 'DESC'
        },
        {
            id: 'sortItem.priceAsc',
            text: 'Price: Low to High',
            attribute: 'price',
            sortDirection: 'ASC'
        },
        {
            id: 'sortItem.priceDesc',
            text: 'Price: High to Low',
            attribute: 'price',
            sortDirection: 'DESC'
        }
```

In this object, only `price` has sorting for both ASC and DESC and other objects were not consistent in providing ASC/DESC values. I decided to add both values to the object, but we might wanna turn the strings returned from it a little bit better.